tests/load/script_archive_unused/deploy_with_genlayer_sdk.py

```
@@ -0,0 +1,367 @@
+            'gas': 0xffffffff,  # Max gas
+            'to': to_checksum_address("0xb7278a61aa25c888815afc32ad3cc52ff24fe575"),  # ConsensusMain
+            'value': 0,
+            'data': tx_data,
+            'chainId': 61999  # GenLayer chain ID
+        }
+        
+        signed_tx = self.account.sign_transaction(transaction)
+        raw_tx = to_hex(signed_tx.raw_transaction)
+        
+        # Send transaction
+        print(f"Sending deployment transaction...", file=sys.stderr)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)